modules/php/Models/Scoresheets/Scoresheet4.php

```diff
@@ -58,27 +58,35 @@ public static function phase5Check(): void {}
 
   public function getScribbleReactions(Scribble $scribble, string $methodSource): array
   {
+    $reactions = [];
+
     $slot = $scribble->getSlot();
-    if (isset($this->linkedResources[$slot])) {
+    if (isset($this->linkedResources[$slot]) && $scribble->getNumber() !== NUMBER_X) {
```

Comment:
Why would you condition this to not writing an X ?
Also, preferred syntax IMO :
$circle = $this->linkedResources[$slot] ?? null;
if(!is_null($circle)){
...

Reply:
Or maybe if(isset($circle)) for an even easier to read syntax

Reply:
> Why would you condition this to not writing an X ?
This is what rules say: "When you write down a **number** in a space with a mineral, immediately circle it".
Could you please check with the publisher if we need to distinguish?

Reply:
Found it: "The spaces filled in with X are considered numbered spaces."